py/torch_tensorrt/fx/converters/aten_ops_converters.py

```diff
@@ -25,9 +25,15 @@
 from torch_tensorrt.fx.converters.impl import activation
 from torch_tensorrt.fx.converters.impl.elementwise import trunc_div
 from torch_tensorrt.fx.converters.impl.elementwise import rsqrt
+from torch_tensorrt.fx.converters.impl.elementwise import clamp
 
 _LOGGER: logging.Logger = logging.getLogger(__name__)
 
+
+def or_none(args, i):
```

Comment:
Thoughts on putting this in utils? Would it matter for acc? or would ACC always give you None?

Reply:
Well acc uses kwargs for optional arguments - but this can be put in utils anyway.